modern/src/LoginPage.js

```diff
@@ -87,10 +98,12 @@ const LoginPage = () => {
 
   return (
     <main className={classes.root}>
+      <div className={classes.sidebar}>
+        {!matches && <img src='/logo.svg' alt='Traccar' />}
+      </div>
       <Paper className={classes.paper}>
-        <img className={classes.logo} src='/logo.svg' alt='Traccar' />
-        <form onSubmit={handleLogin}>
-
+        {matches && <img src='/logo_main.svg' alt='Traccar' />}
```

Comment:
Kindly move the image into the form as well

Reply:
Done.